core/modules/qmeta/QMetaMysql.cc

```diff
@@ -817,6 +817,129 @@ QMetaMysql::_checkDb() {
                                value);
     }
 
+    createQueryStatsTmpTable();
```

Comment:
What happens if exception is thrown by this method, entire QMeta becomes unusable? Maybe add try/catch around it and handle this condition. Or rather move all of that stuff to a separate thing independent of QMeta.